src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -87,26 +95,18 @@ def compress(self):
         torch.nn.Module
             model with specified modules compressed.
         """
-        if self.modules_wrapper is not None:
-            # already compressed
-            return self.bound_model
-        else:
-            self.modules_wrapper = []
-
-        modules_to_compress = self.detect_modules_to_compress()
-        for layer, config in modules_to_compress:
-            wrapper = self._wrap_modules(layer, config)
-            self.modules_wrapper.append(wrapper)
-
-        self._wrap_model()
         return self.bound_model
```

Comment:
how about using `raise NotImplementedError()` here?

Reply:
Quantizer does not override compress method